test/__snapshots__/standalone-single-spa.test.js.snap

```diff
@@ -42,6 +40,8 @@ exports[`standalone-single-spa-webpack-plugin basic-parcel 1`] = `
         singleSpa.mountRootParcel(parcelConfig, { domElement: parcelContainer, name: 'basic-parcel' });
       });
     </script>
+  </head>
+  <body>
```

Comment:
this snapshot changes seem... unrelated?

Reply:
they were related to updating the devDependency for html-webpack-plugin. APparently v5 has some breaking changes. Since it's just a dev dependency though it doesn't force the change on any users.